script/chart_upstream_checker.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright (c) 2018 Bitnami
```

Comment:
```suggestion
# Copyright (c) 2021 Bitnami
```

Reply:
I did change it, trying to be more aligned with other similar projects [1] (i.e., using SPDX and something like _the Kubeapps contributors._), but I noticed it would require a further discussion. This change should propagate across the rest of the files in our project.
But, while undoing the changes, it got changed back to 2018 :facepalm: 
Thanks for catching it!

[1] https://github.com/vmware-tanzu/pinniped/blob/a8bccc5432a1aae7e48cd931dd1b31b66049f3ea/hack/check-copyright-year.sh#L3-L4